### DISCUSSION TITLE
The minimal project needs to be updated or explained better

### DISCUSSION THREAD

**Discussion by OOaker** [DaveGreen-Games/ShapeEngine#37] [Q&A]
Sure I can fix it myself but we want people to be able to easily start out. I can't tell you how many times I myself have instantly closed and deleted a solution I downloaded because it just didn't work right out of the box. When I try something new troubleshooting is not the first thing I want to do.
![image](https://github.com/user-attachments/assets/ca309a68-429b-4c17-b0c1-5f63b4f9fc74)

**Comment by SoloByte** [comment]
Did you use the nuget package or downloaded it from source (main) ?

The readme is out of sync because I just merged some changes and I have to update it.

The demo minimal project setup will be out of sync until I publish a new nuget version.

**Comment by SoloByte** [comment]
This is the new version.

```c#
public static class Program
{
    public static void Main(string[] args)
    {
        var game = new MyGameClass(GameSettings.StretchMode, WindowSettings.Default);
	game.Run();
    }
}
public class MyGameClass : Game
{
    public MyGameClass(GameSettings gameSettings, WindowSettings windowSettings) : base(gameSettings, windowSettings) { }
    protected override void DrawGame(ScreenInfo game)
    {
        game.Area.Draw(new ColorRgba(Color.DarkOliveGreen));
        game.Area.DrawLines(12f, new ColorRgba(Color.AntiqueWhite));
        game.MousePos.Draw(24f, new ColorRgba(Color.Lime), 36);
    }
}
```
